tests/test_kafka_consumer.py

```diff
@@ -141,7 +141,9 @@ def test_parse_organization_message():
             "created_at": "2014-04-17T18:21:09",
             "orga_sp": 1,
             "followers": 357,
-            "datasets": 56
+            "datasets": 56,
+            "views": 42,
```

Comment:
You can add an assert that these field is 2p-normalized.

Reply:
We said we wouldn't use `log2p` on those fileds no?